ocaml/xenopsd/xc/service.ml

```diff
@@ -504,7 +514,15 @@ module Vgpu = struct
       @ device_args
     in
     let fd_arg = if restore then ["--resume"] else [] in
-    List.concat [base_args; fd_arg]
+    (* support for NVidia VCS (compute) vGPUs *)
+    let no_console =
+      match List.exists is_compute_vgpu vgpus with
```

Comment:
Hmm... I'd expect to not want a console if _all_ vGPUs are compute ones, not just one of them. Is that right?

Reply:
I have to ask @xennifer. Could switch to `List.for_all`.

Reply:
I've changed it to `for_all` after talking to @xennifer